.github/renovate.json

```diff
@@ -1,14 +1,16 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "postUpgradeTasks": {
-    "commands": [
-      "cp parts/linux/cloud-init/artifacts/components.json parts/linux/cloud-init/artifacts/components-N-1.json"
-    ],
-    "fileFilters": [
-      "parts/linux/cloud-init/artifacts/components.json"
-    ],
-    "executionMode": "branch"
-  },
+  "enabledManagers": ["regex"],
+  //ONLY SUPPORTED WITH LOCAL RUNNER
```

Comment:
does renovate support in-line comments in JSON? this isn't usually valid syntax

Reply:
same question here